imu_filter_madgwick/launch/imu_filter_madgwick.launch

```diff
@@ -0,0 +1,27 @@
+<!-- Node for merging magnetometer and accelerometer data into a single imu message -->
+<launch>
+  #### Nodelet manager ######################################################
+
+  <node pkg="nodelet" type="nodelet" name="imu_manager" 
+    args="manager" output="screen" />
+
+  #### IMU Driver ###########################################################
+
+  <node pkg="nodelet" type="nodelet" name="ImuFilterNodelet" 
+    args="load imu_filter_madgwick/ImuFilterNodelet imu_manager" 
+    output="screen">
+
+    # supported data rates: 4 8 16 24 32 40 ... 1000 (in ms)
+    <param name="use_magnetic_field_msg" value="true"/>
+
```

Comment:
The parameter use_magnetic_field_msg was removed long time ago, so this line should be removed as well.